tests/testCoaddApCorrMap.py

```diff
@@ -125,34 +118,29 @@ def assertApCorrMap(self, apCorrMap, pointList):
         for i, point in enumerate(pointList):
             weights = [i+1, i+2]
             values = [i+1, i+2]
```

Comment:
Here and throughout there are many cases where spaces need to be added around the + and - operators, or removed around the * and / operators.